oauth2/pom.xml

```diff
@@ -91,7 +91,17 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Version>${project.version}</Bundle-Version>
+                        <Import-Package>
+                            !org.eclipse.microprofile.*,
+                            org.eclipse.microprofile.config;version="[1.0,3)",
+                            *
+                        </Import-Package>
+                        <Export-Package>
+                            !org.eclipse.microprofile.*,
+                            fish.payara.security.oauth2.*
+                        </Export-Package>
                     </instructions>
+                    <excludeDependencies>microprofile,microprofile-config-api</excludeDependencies>
```

Comment:
Are all of these necessary? In theory only `*` and the rule for config package should be necessary, and only for imports.
Exports and excludeDependencies are not needed (I didn't try though).

Reply:
You're right. In the end it's enough to place the config package in import before the microprofile and * imports. When I added it to the end of the list, it was being ignored because the packages were handled by the previous rules.